lib/config.js

```diff
@@ -7,30 +7,31 @@ var log = require('npmlog')
 var mkdirp = require('mkdirp')
 
 module.exports = function (options) {
-  var projectPath = options.path || process.cwd()
+  var projectPath = process.cwd()
 
   var pkg = require(path.join(projectPath, 'package.json'))
 
   var config = {
     name: pkg.name,
     paths: {
-      data: options.data || path.join(projectPath, 'data'),
-      project: projectPath,
-      www: options.www || path.join(projectPath, 'www')
+      data: options.data || path.join(projectPath, '.hoodie'),
+      public: options.public || path.join(projectPath, 'public')
     },
-    db: {}
+    db: {},
+    admin: {},
+    account: options.account || {}
   }
 
-  var wwwExists
+  var publicFolderExists
   try {
-    wwwExists = fs.statSync(config.paths.www).isDirectory()
+    publicFolderExists = fs.statSync(config.paths.public).isDirectory()
   } catch (err) {
-    wwwExists = false
+    publicFolderExists = false
   }
 
-  if (!wwwExists) {
-    log.info('config', 'The "www" app path does not exist. Using default Hoodie app.')
-    config.paths.www = path.dirname(require.resolve('my-first-hoodie'))
+  if (!publicFolderExists) {
+    log.info('config', 'The "public" app path does not exist. Using hoodie-server/public.')
+    config.paths.public = path.resolve(__dirname, '../public')
```

Comment:
the `www` folder has been renamed to `public` as per https://github.com/hoodiehq/hoodie-server/issues/436